test/tests/api/v1_1/discovery_tests.py

```diff
@@ -35,29 +36,29 @@ def __get_workflow_status(self, id):
     @test(groups=['test_discovery_post_reboot'], depends_on_groups=["test-node-poller"])
     def test_node_workflows_post_reboot(self):
```

Comment:
It's here to post this new workflow
https://github.com/RackHD/RackHD/pull/232/files#diff-ecb0f933ac478b58493da50b899c920aR58